### PR TITLE
Document featurehash

### LINKED ISSUE #363: [apidoc] The ngeo.format.FeatureHash methods have no descriptions
The methods of `ngeo.format.FeatureHash` have no descriptions. See http://camptocamp.github.io/ngeo/master/apidoc/ngeo.format.FeatureHash.html.

### PR DESCRIPTION
**Pull request by ger-benjamin**
Fix #363

### CODE REVIEW

File: src/ol-ext/format/featurehash.js [hunk 1]
```diff
@@ -52,11 +52,11 @@ ngeo.format.FeatureHashStyleTypes_[ol.geom.GeometryType.MULTI_POLYGON] =
...
 /**
- * This file provides an OpenLayers format for encoding
- * and decoding features for use in permalinks.
+ * @classdesc
+ * Provide an OpenLayers format for encoding and decoding features for use
+ * in permalinks.
  *
- * The code is based on Stéphane Brunner's URLCompressed format:
- * <https://github.com/sbrunner/OpenLayers-URLCompressed>
+ * The code is based on Stéphane Brunner's URLCompressed format.
```
Comment (sbrunner) [inline src/ol-ext/format/featurehash.js]: Why do you remove the URL ?

Comment (sbrunner) [inline src/ol-ext/format/featurehash.js]: I just see that you move it to @see :-)

File: src/ol-ext/format/featurehash.js [hunk 2]
```diff
@@ -337,6 +357,8 @@ ngeo.format.FeatureHash.encodeStyleText_ = function(textStyle, encodedStyles) {
 
 
 /**
+ * Read a logical sequence of characters and return a corresponding
+ * ol.geom.LineString.
```
Comment (sbrunner) [inline src/ol-ext/format/featurehash.js]: use `{@link ol.geom.LineString}` it' possible to make it working by modifying the template.
also bellow...

**Comment by sbrunner** [comment]
Except a small comment that's looks good for me :-)

**Comment by fredj** [comment]
+1

**Pull request merged**